pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -1293,6 +1299,28 @@ func setJvmDefault(jvm *api.JvmObject) {
 	}
 }
 
+func setKafkaExtDefault(kafka *api.KafkaExt) {
+	if kafka != nil {
+
+		if kafka.BootstrapServers == "" {
+			//if aren't present bootstrap servers we consider a disabled kafka extension
+			kafka.ExtDisabled = constants.KIE_KAFKA_EXT_DISABLED_DEFAULT
+			kafka.BootstrapServers = constants.KIE_KAFKA_EXT_BOOTSTRAP_SERVERS_DEFAULT
+		}
+
+		if kafka.MaxBlockMs == 0 {
+			kafka.MaxBlockMs = constants.KIE_KAFKA_EXT_MAX_BLOCK_MS_DEFAULT
+		}
+
+		if kafka.GroupID == "" {
```

Comment:
for strings it is better to use len(xxx) == 0 or < 0..

Reply:
ok

Reply:
ok